### ISSUE TITLE
Internal Error after activation of redirection plugin in latest php version.

### ISSUE THREAD

**Issue by jaidevsharma** [johngodley/redirection#2762]
### Summary
Migrated zzorganic.com to the latest version of php using AllInOneWPMigration plugin.
When redirection is activated, we get the following error when we try to edit any page.
-------
Internal Server Error
The server encountered an internal error or misconfiguration and was unable to complete your request.

Please contact the server administrator at you@example.com to inform them of the time this error occurred, and the actions you performed just before this error.

More information about this error may be available in the server error log.
----
Server Log is as follows:

[Thu Feb 11 16:52:13.150961 2021] [substitute:error] [pid 14118:tid 139881145677568] [clie
nt 67.184.130.93:54431] AH01328: Line too long, URI /wp-admin/post.php, referer: https://z
zorganic.com/wp-admin/edit.php?post_type=page
[Thu Feb 11 16:52:13.158901 2021] [proxy_fcgi:error] [pid 14118:tid 139881145677568] (12)C
annot allocate memory: [client 67.184.130.93:54431] AH01075: Error dispatching request to 
: (passing brigade to output filters), referer: https://zzorganic.com/wp-admin/edit.php?po
st_type=page
[Thu Feb 11 17:02:49.333381 2021] [substitute:error] [pid 14118:tid 139881019852544] [clie
nt 67.184.130.93:54556] AH01328: Line too long, URI /wp-admin/post.php, referer: https://z
zorganic.com/wp-admin/edit.php?post_type=page
[Thu Feb 11 17:02:49.335078 2021] [proxy_fcgi:error] [pid 14118:tid 139881019852544] (12)C
annot allocate memory: [client 67.184.130.93:54556] AH01075: Error dispatching request to
: (passing brigade to output filters), referer: https://zzorganic.com/wp-admin/edit.php?po
st_type=page
-------

### Environment
bitnami- Linux instance (latest version) via Amazon Lightsail.

Redirection- Version 5.0.1 | By John Godley 
Wordpress- v5.6.1

PHP 7.4.13 (cli) (built: Dec  1 2020 10:42:31) ( NTS )
Copyright (c) The PHP Group
Zend Engine v3.4.0, Copyright (c) Zend Technologies
    with Zend OPcache v7.4.13, Copyright (c), by Zend Technologies

NOTE: If we deactivate the Redirection plugin and try to edit the file, it works without error. We did not have any edit issues in the older instance which used older php versions.

**Comment by johngodley** [comment]
Can you narrow down the problem that you think Redirection is causing? I don't really know what you are showing me above or how it relates to Redirection.

**Comment by jaidevsharma** [comment]
We were getting the above errors when we migrated the website to php7.4.13. So we started to deactivate the plugins. When we deactivate redirection plugin, we don't get the errors given above and are able to edit our files. So it seems that the presence of the redirection plugin is causing the above problem. We do not know how redirection is causing this- were hoping if someone can give us some insight on what we can do to track this further. It looks like php7.4.13 was available since Dec1, so we are not sure if the plugin has been tested with the latest php. Also, maybe this is something you have seen before - the main errors seem to be "cannot allocate memory" and "line too long"- there might be some setting interfering with redirection. Any insight will be helpful.
